forage-android/src/main/java/com/joinforage/forage/android/ecom/services/vault/bt/BasisTheoryPinSubmitter.kt

```diff
@@ -56,7 +52,7 @@ internal class BasisTheoryPinSubmitter(
         val proxyRequest: ProxyRequest = ProxyRequest().apply {
             headers = vaultProxyRequest.headers
             body = ProxyRequestObject(
-                pin = foragePinEditText.getTextElement(),
+                pin = foragePinEditText.getTextElement() as TextElement,
```

Comment:
why is this casting now necessary?

Reply:
Good question! I put a comment near [the method definition](https://app.graphite.dev/github/pr/teamforage/forage-android-sdk/248/Increase-flexibility-of-ForageElement-hierarchy?panel=files#comment-PRRC_kwDOIQqcBs5fM2HH) for context. But the tl;dr is:
* `getTextElement()` returns a `View`
* `BasisTheoryPinSubmitter` will only ever be invoked when the backing PIN input is BT's `TextElement`
* So, we can safely cast here, to bridge to pass BT what it expects while also satisfying the compiler in `ForagePinElement`

Lmk whether that clarifies